Parsers/ASimFileEvent/Parsers/ASimFileEventMicrosoftSecurityEvent.yaml

```diff
@@ -0,0 +1,92 @@
+ParserName: ASimFileEventMicrosoftSecurityEvents
+EquivalentBuiltInParser: _ASim_FileEvent_MicrosoftSecurityEvents
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let Parser=(disabled:bool=false)
+  {
+  let EventTypeLookup = datatable (AccessMask:string,EventType:string)
+  [
+      "0x1", "ObjectAccessed"
+      , "0x10", "MetadataModified"
+      , "0x100", "MetadataModified"
+      , "0x10000", "ObjectDeleted"
+      , "0x2", "ObjectModified"
+      , "0x20000", "MetadataAccessed"
+      , "0x4", "ObjectModified"
+      , "0x40", "ObjectDeleted"
+      , "0x40000", "MetadataModified"
+      , "0x6", "ObjectModified"
+      , "0x8", "MetadataAccessed"
+      , "0x80", "MetadataAccessed"
+      , "0x80000", "MetadataModified"
+  ];
+  let UserTypeLookup = datatable (AccountType:string, ActorUserType:string)
+  [
+    'User', 'Regular',
+    'Machine', 'Machine'
+  ];    
+  let KnownSIDs = datatable (sid:string, username:string, type:string)
+  [
+    'S-1-5-18', 'Local System', 'Simple',
+    'S-1-0-0', 'Nobody', 'Simple'
+  ];
+  SecurityEvent
+  | where not(disabled)
+  | where EventID == 4663 
+    and ObjectType == "File"
+    and ObjectName !startswith @"\Device\"
+  | project TimeGenerated, EventID, AccessMask, ProcessName, SubjectUserSid, AccountType, Computer, ObjectName, ProcessId, SubjectUserName, SubjectAccount, SubjectLogonId, HandleId
+  | lookup EventTypeLookup on AccessMask
+  | lookup UserTypeLookup on AccountType
+  | lookup KnownSIDs on $left.SubjectUserSid == $right.sid
+  | extend ActingProcessName = ProcessName
+    , ActorUsername = iff (SubjectUserName == "-", username, SubjectAccount)
+    , ActorUsernameType = iff(SubjectUserName == '-',type, 'Windows')
+    , EventStartTime = TimeGenerated
+    , EventEndTime = TimeGenerated
+    , TargetFilePath = ObjectName
+    , TargetFilePathFormat = "Windows Local"
+    , ActingProcessId = tostring(toint(ProcessId))
+    , EventOriginalType = tostring(EventID)
+    , ActorUserIdType="SID"
+    , TargetFilePathType="Windows Local"
+    , Type=type
+  | project-away EventID, ProcessId, AccountType, username
+  | project-rename ActorUserId = SubjectUserSid
+    , DvcHostname = Computer
+    , Process = ProcessName
+    , FilePath = ObjectName
+    , ActorSessionId = SubjectLogonId
+    , FileSessionId = HandleId
+  | extend EventSchema = "FileEvent"
+    , EventSchemaVersion = "0.1.1"
+    , EventResult = "Success"
+    , EventCount = int(1)
+    , EventVendor = 'Microsoft'
+    , EventProduct = 'Security Events'
+    , Dvc = DvcHostname
+    , ActorWindowsUsername = ActorUsername
+    , User = ActorUsername
+    , ActorUserSid = ActorUserId
+    | project-away AccessMask,ActorWindowsUsername,FileSessionId,SubjectAccount,SubjectUserName,TargetFilePathFormat
```

Comment:
@Alekhya0824 please project-away this too
![image](https://github.com/user-attachments/assets/192e358c-1ea0-41bd-b441-2e93957eaf1f)

Reply:
done